Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -1601,6 +1601,17 @@ class ByteBufferTest: XCTestCase {
 
         XCTAssertEqual(0, buf.readableBytes)
     }
+
+    func testPossiblyLazilyBridgedString() {
```

Comment:
Does the test verify all three different code-paths that we have now ?

Reply:
@normanmaurer on correctly functioning toolchains we can't hit all three. But on Mac we hit the first 2 and on Linux we'll also hit the last one (at the moment but that'll be fixed).